test/utils/xcm.ts

```diff
@@ -510,6 +510,58 @@ export class XcmFragment {
         return this;
     }
 
+    deposit_asset_definite(
+        location: any,
+        amount: bigint,
+        beneficiary: `0x${string}`,
+        network: XcmV3JunctionNetworkId["type"] | null = null
+    ): this {
+        if (hexToU8a(this.config.beneficiary).length === 20) {
```

Comment:
Theoretically, if we pass any number of bytes besides 20, it will think that it is 32. We may want to add validation: if not 20 or not 32 -> throw error.

Reply:
Also, I think we can make it a bit more compact:
```suggestion 
const keyBytes = hexToU8a(this.config.beneficiary);

    const beneficiaryInterior =
        keyBytes.length === 20
            ? { AccountKey20: { network, key: beneficiary } }
            : { AccountId32: { network, key: beneficiary } };

    this.instructions.push({
        DepositAsset: {
            assets: {
                Definite: [
                    {
                        id: { Concrete: location },
                        fun: { Fungible: amount },
                    },
                ],
            },
            beneficiary: {
                parents: 0,
                interior: { X1: beneficiaryInterior },
            },
        },
    });

    return this;

 ```